docs/community/governance.md

```diff
@@ -73,28 +73,82 @@ long experience with both the project and the larger ecosystem. When the core
+
+The SC membership, including the Institutional and Funding Partner seat, is revisited every January.
+SC members who do not actively engage with the SC duties are expected to resign. New members for 
+vacant spots are added by nomination by a core developer. Nominees should have demonstrated
 long-term, continued commitment to the project and its [mission and values](mission-and-values). A
 nomination will result in discussion that cannot take more than a month and
 then admission to the SC by consensus. During that time deadlocked votes of the SC will
-be postponed until the new member has joined and another vote can be held.
-
-The napari steering council may be contacted at `napari-steering-council@googlegroups.com`.
-Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team.
+be postponed until the new member has joined and another vote can be held. The Institutional and Funding Partner seat
+is elected by the Institutional and Funding Partner Advisory Council.
+
+The SC may be contacted at `napari-steering-council@googlegroups.com`.
+Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team. For a list of the current steering council see our [About Us](team) page.
+
+### Institutional and funding partners
+
+The SC is the primary leadership body for napari. No outside institution,
+individual or legal entity has the ability to own or control the project
+other than by participating in napari as contributors, core developers, and
+SC members. However, because institutions can be an important source of
+funding and contributions for the project, it is important to formally
+acknowledge institutional participation in the project. We call institutions
+recognized in this way Institutional and Funding Partners.
+
+Institutions become eligible to become an Institutional Partner by employing
+individuals who actively contribute to napari as part of their official
+duties, or by committing significant funding to napari, as determined by the
+SC. Once an institution becomes eligible to become an Institutional Partner,
+the SC must nominate and approve the institution as an Institutional and
+Funding Partner. At that time one individual from the Institutional and
+Funding Partner is expected to become the Institutional and Funding Partner
+Representative and serve on the Institutional and Funding Partner Advisory
+Council. The role of the Institutional and Funding Partner Advisory Council
+is to provide input on project directions and plans, and to elect one
+Institutional and Funding Partner Representative to hold the Institutional
+and Funding Partner seat on the SC.
+
+The Institutional and Funding Partners Advisory Council is expected to
+self-organize according to rules agreed to by the existing Institutional
+Funding Partners. This document does not prescribe how Institutional Funding
```

Comment:
given that the SC have no oversight here, the SC/wider community should probably review these terms during the instantiation of the first IFP to ensure everything seems reasonable from the outset 🙂

Reply:
Right; should we require/expect such organization to happen somewhere in the open?

Reply:
:shrug: I dunno. I actually added this explicitly because I'd like to not be concerned here — if they want to make their decisions through gladiatorial-style combat, that's fine :joy:, as long as they send us a representative that actually represents them. :joy: My *only* concern is if some orgs wield their power and influence too haphazardly, other orgs might feel disenfranchised. But that's the "good faith" clause and I hope that as long as we have various communications channels (and there *is* an IFP council and the SC will meet with that council regularly), then we should be ok?

But, the idea of mandating transparency is a good one. Do either of you want to suggest some wording around it?

Reply:
As long as the gladiatorial combat is open to everyone I'm on board 😆 I'm not sure... we could just have a zulip channel dedicated to such discussions, though probably face-to-face meetings are better for this sort of decision.

Reply:
I think having the SC review when the first IFP comes on is a good idea - i'm not sure that needs to be formally captured in governance as it will only happen once. I'm imagining that a good-faith approach would somehow incorporate the amount of support each partner is giving - but is it exactly a weighted sum of employees and grant dollars or something a bit more nuanced I think can be deferred to later

Reply:
Is there only one "Institutional and Funding Partner Advisory Council" which all of the IFPs contribute to? Or does each IFP have their own Advisory Council and each elects a representative to sit on the SC?

Reply:
Yes there is only one IFP Advisory council with a single representative from each IFP. They are expected to elect that single seat for the SC

Reply:
> But, the idea of mandating transparency is a good one. Do either of you want to suggest some wording around it?

I have added something